frame/contracts/src/lib.rs

```diff
@@ -203,8 +203,15 @@ pub type AliveContractInfo<T> =
 pub struct RawAliveContractInfo<CodeHash, Balance, BlockNumber> {
 	/// Unique ID for the subtree encoded as a bytes vector.
 	pub trie_id: TrieId,
-	/// The size of stored value in octet.
+	/// The total number of bytes used by this contract.
+	///
+	/// It is a sum of each key-value pair stored by this contract.
 	pub storage_size: u32,
+	/// The number of key-value pairs that have values of zero length.
+	/// The condition `empty_pair_count ≤ total_pair_count` always holds.
+	pub empty_pair_count: u32,
+	/// The total number of key-value pairs in storage of this contract.
+	pub total_pair_count: u32,
```

Comment:
What to we need this value for? I see that it is updated but never used for anything. Only the empty values are used.

Reply:
Well noticed! 

I figured that since we are counting `empty_pair_count` it might be useful to collect the total pair count in case we (or the governance) find itself in need of the number of non-empty pairs. For example, charge a constant fee per a kv pair. 

If we don't introduce it here they (the governance) would need to painfully collect and submit that info should the need arise. OTOH, the difference in performance should be negligible and the logic seems to be straightforward.

However, I don't feel strong about this. I can roll this back as well if you think that is not a good reason.

Reply:
From a runtime standpoint is is negligible of course. I was thinking about storage overhead. But I think that is also OK when compared to all the other data that is stored. I am OK with keeping it.